packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -26,6 +26,12 @@ export interface IBasePickerState {
   selectedIndices?: number[];
 }
 
+export const enum PickerAriaIdEnum {
```

Comment:
>PickerAriaIdEnum [](start = 18, length = 16)

Just squinting at the code, why does this need to be exported?

Reply:
For anything that want's to subclass.

Reply:
I'm still confused - how are the sub classes using this? 

It still seems you could still also avoid the enum entirely (it has bundle size impact). If really using an enum, you should give each enum property an explicit value (e.g. = 0, = 1, etc.)